### PR TITLE
Booga Booga the FOB drone has a light now so you can see things.

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
AI couldn't see shit using the drone. This fixes that in a simple way. 

## Why It's Good For The Game
Seeing good.

## Changelog
:cl:
fix: fix the AI being blind while using the FOB drone.
/:cl:

### CODE REVIEW

File: code/modules/mining/remote_fob/remote_fob_fobdrone.dm [hunk 1]
```diff
@@ -17,6 +17,7 @@ GLOBAL_LIST_INIT(blocked_remotebuild_objs, typecacheof(list(/obj/machinery/compu
 /mob/camera/aiEye/remote/fobdrone/Initialize()
 	. = ..()
 	starting_area = get_area(loc)
+	set_light(3)
```
Comment (BraveMole) [inline code/modules/mining/remote_fob/remote_fob_fobdrone.dm:20]: use set_light_on, set_light is meant for static lights. That's just a performance issue

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/mining/remote_fob/remote_fob_fobdrone.dm (hunk 1 above)
Comment (lbnesquik) [reply code/modules/mining/remote_fob/remote_fob_fobdrone.dm:20]: Sure. Will do. I'll make it a little more potent too as 3 is actually like PDA tier.

File: code/modules/mining/remote_fob/remote_fob_fobdrone.dm [hunk 2]
```diff
@@ -10,13 +10,17 @@ GLOBAL_LIST_INIT(blocked_remotebuild_objs, typecacheof(list(/obj/machinery/compu
...
+	light_range = 5
+	light_power = 4
 
 	var/area/starting_area
 	var/turf/spawnloc
 
 /mob/camera/aiEye/remote/fobdrone/Initialize()
 	. = ..()
 	starting_area = get_area(loc)
+	set_light_on(TRUE)
```
Comment (TiviPlus) [inline code/modules/mining/remote_fob/remote_fob_fobdrone.dm:23]: This can just be light_on = TRUE On the definition

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by TiviPlus** [comment]
I ssaid definition not initialize

File: code/modules/mining/remote_fob/remote_fob_fobdrone.dm [hunk 3]
```diff
@@ -10,13 +10,17 @@ GLOBAL_LIST_INIT(blocked_remotebuild_objs, typecacheof(list(/obj/machinery/compu
...
+	light_range = 5
+	light_power = 4
 
 	var/area/starting_area
 	var/turf/spawnloc
 
 /mob/camera/aiEye/remote/fobdrone/Initialize()
 	. = ..()
 	starting_area = get_area(loc)
+	light_on = TRUE
```
Comment (BraveMole) [inline code/modules/mining/remote_fob/remote_fob_fobdrone.dm:23]:
```suggestion
```

File: code/modules/mining/remote_fob/remote_fob_fobdrone.dm [hunk 4]
```diff
@@ -10,13 +10,17 @@ GLOBAL_LIST_INIT(blocked_remotebuild_objs, typecacheof(list(/obj/machinery/compu
 	icon_state = "drone"
 	use_static = FALSE
 	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	light_system = MOVABLE_LIGHT
+	light_range = 5
+	light_power = 4
```
Comment (BraveMole) [inline code/modules/mining/remote_fob/remote_fob_fobdrone.dm:15]:
```suggestion
	light_power = 4
	light_on = TRUE
```

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**